Makefile

```diff
@@ -19,17 +19,17 @@ test:	## Run test suite
 test/coverage:	## Run test suite with coverage report
 	go test -v ./... -cover
 
-build:	## Build operator Docker image
-	@docker build -f Dockerfile --build-arg operator_version=$(curVersion) -t $(DOCKER_IMG_TAG) .
+build/operator:	## Build operator Docker image
```

Comment:
Forgot to mention - you'll want to update the `.PHONY:` definitions up top with these updated makefile targets. That way the makefile won't try searching for whether these files fake files exist 😄